infrastructure/resource_group_level/modules/dtfs-central-api.bicep

```diff
@@ -0,0 +1,169 @@
+param location string
+param environment string
+param containerRegistryName string
+param appServicePlanEgressSubnetId string
+param appServicePlanId string
+param privateEndpointsSubnetId string
+param cosmosDbAccountName string
+param cosmosDbDatabaseName string
+param logAnalyticsWorkspaceId string
+
+var dockerImageName = '${containerRegistryName}.azurecr.io/dtfs-central-api:${environment}'
+var dockerRegistryServerUsername = 'tfs${environment}'
+
+@secure()
+param secureSettings object = {
+  
+}
+
+// These values are taken from an export of Configuration on Dev (& validating with staging).
+@secure()
+param additionalSecureSettings object = {
+  DOCKER_REGISTRY_SERVER_PASSWORD: 'test-value'
+}
+
+// These values are hardcoded in the CLI scripts
+var settings = {
+  WEBSITE_DNS_SERVER: '168.63.129.16'
+  WEBSITE_VNET_ROUTE_ALL: '1'
+  PORT: '5000'
+  WEBSITES_PORT: '5000'
+}
+
+// These values are taken from an export of Configuration on Dev (& validating with staging).
+var additionalSettings = {
+  APPLICATIONINSIGHTS_CONNECTION_STRING: applicationInsights.properties.ConnectionString
+  // Note that the config has APPINSIGHTS_INSTRUMENTATIONKEY as a slot setting, though the advice
+  // is not to use at the same time as its replacement - APPLICATIONINSIGHTS_CONNECTION_STRING
+  // APPINSIGHTS_INSTRUMENTATIONKEY: applicationInsights.properties.InstrumentationKey
+  DOCKER_ENABLE_CI: 'true'
+  DOCKER_REGISTRY_SERVER_URL: '${containerRegistryName}.azurecr.io'
+  DOCKER_REGISTRY_SERVER_USERNAME: dockerRegistryServerUsername
+  LOG4J_FORMAT_MSG_NO_LOOKUPS: 'true'
+  TZ: 'Europe/London'
+  WEBSITE_HTTPLOGGING_RETENTION_DAYS: '3'
+  WEBSITES_ENABLE_APP_SERVICE_STORAGE: 'false'
+}
+
+var nodeEnv = environment == 'dev' ? { NODE_ENV: 'development' } : {}
+
+var appSettings = union(settings, secureSettings, additionalSettings, additionalSecureSettings, nodeEnv)
+
+var dtfsCentralApiName = 'tfs-${environment}-dtfs-central-api'
+var privateEndpointName = 'tfs-${environment}-dtfs-central-api'
+var applicationInsightsName = 'tfs-${environment}-dtfs-central-api'
+
+
+// These have come from the CLI scripts
+var mongoDbConnectionString = replace(cosmosDbAccount.listConnectionStrings().connectionStrings[0].connectionString, '&replicaSet=globaldb', '')
+var connectionStrings = {
```

Comment:
Note that in reference-data-proxy (now named `external-api`) the mongodb connection string is sent via normal config. I'm not sure why there is the inconsistency. The code does say:
```
        # NOTE: MULESOFT envs were producing an error when used in connection_string so have been moved to appsettings
        # ref: https://docs.microsoft.com/en-us/azure/app-service/configure-common
```

Reply:
ok cool. let's get them standardised then